src/api/handlers/charts.go

```diff
@@ -20,6 +20,7 @@ func GetChart(params operations.GetChartParams, c data.Charts) middleware.Respon
 		return notFound(chartResourceName)
 	}
 	chartResource := helpers.MakeChartResource(chart, params.Repo)
+	helpers.AddLatestLinks(chartResource, *chart.Version)
```

Comment:
I thought that the latest link was going to be accessible via `/charts` and `/charts/{repo}` which return a non-versioned Chart resource no?

Basically only removing it from /charts/{repo}/{chartName}/versions